crates/tuono/src/route.rs

```diff
@@ -30,7 +30,7 @@ impl AxumInfo {
         let module_import = module
             .as_str()
             .to_string()
-            .replace('/', "_")
+            .replace(['/', '-'], "_")
```

Comment:
We should avoid this. I think this means that: `route/subroute` and `route-subroute` would get mapped with the same value. I'd follow the same pattern as the `.`: `_hyphen_`